src/osx/carbon/font.cpp

```diff
@@ -659,6 +691,16 @@ CTFontRef wxFont::OSXGetCTFont() const
     return (CTFontRef)(M_FONTDATA->m_ctFont);
 }
 
+CFDictionaryRef wxFont::OSXGetCTFontAttributes() const
+{
+    wxCHECK_MSG( M_FONTDATA != NULL , 0, wxT("invalid font") );
```

Comment:
Very minor, but this `0` should be `NULL` here, for consistency -- we use `NULL`s for other `XXXRef`s elsewhere.